__tests__/articles.tests.js

```diff
@@ -99,14 +99,21 @@ describe('GET /api/articles/:article_id', () => {
             body: expect.any(String),
             created_at: expect.any(String),
             votes: expect.any(Number),
-            article_img_url: expect.any(String)
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number)
```

Comment:
This is a new behaviour we are adding so ideally this should be added in a new test instead of changing a previous test.